tests/basictests.cpp

```diff
@@ -506,6 +539,11 @@ namespace parse_api_tests {
     for (auto [doc, error] : parser.load_many(AMAZON_CELLPHONES_NDJSON)) {
       if (error) { cerr << error << endl; return false; }
       if (!doc.is<dom::array>()) { cerr << "Document did not parse as an array" << endl; return false; }
+      auto arr = doc.get<dom::array>().value(); // let us get the array
```

Comment:
Was this because `size()` wasn't implemented on simdjson_result<dom::array>? It's fine to file a bug for later, but we should probably do that; we already implement arrray iterator on both simdjson_result<dom::array> and simdjson_result<dom::element>.

Reply:
> Was this because size() wasn't implemented on simdjson_resultdom::array? 

Yes. Fixed in a commit.